.github/workflows/build.yaml

```diff
@@ -22,34 +22,47 @@ concurrency:
 # will skip running `yarn install` if it successfully fetched from cache
 
 jobs:
-  fmt:
+  prettier:
     name: Format with Prettier
     runs-on: ubuntu-latest
     timeout-minutes: 5
     steps:
       - name: Checkout repo
         uses: actions/checkout@v3
 
+      - name: Run prettier with actionsx/prettier
+        uses: actionsx/prettier@v2
+        with:
+          args: --check --loglevel=warn .
```

Comment:
The downside to this action is we can't specify the version of Prettier. It's significantly faster than our previous Prettier step so I think that's worth switching. Happy to change back and use something like `npx` if desired.

Reply:
I think this is OK, probably there is not a huge amount of difference in Prettier versions and we do not change code-server code very often anyway.

Long term it does seem worth somehow making it use the same version since this has the potential to be frustrating since every time you push to CI it is more or less a gamble whether Prettier passes.

Reply:
Maybe what we should do is pin our package.json version to whatever version the action uses.  That way we have parity with CI and local.

Reply:
Dude you already did that, I had not gone through the PR yet lol.  Nice!  Works for me.

Reply:
I guess we could still go out of date if the action updates though.  Ah well.

Reply:
I can pin the action too! If the action does update, Rennovate will tell us and then i can update prettier to match

Reply:
Oh snap good point.